pkg/disk/nodeserver.go

```diff
@@ -625,6 +619,10 @@ func (ns *nodeServer) NodeStageVolume(ctx context.Context, req *csi.NodeStageVol
 
 	// Block volume not need to format
 	if isBlock {
+		if err := ns.mounter.EnsureBlock(targetPath); err != nil {
```

Comment:
We must initialize targetPath before using it as a parameter in CheckDeviceAvailable.

Reply:
Why? I don't see anything that requires that path to exist on disk.